planning/planners/src/encode.rs

```diff
@@ -268,8 +268,12 @@ fn enforce_refinement(t: TaskRef, supporters: Vec<TaskRef>, model: &mut Model) {
         // if the supporter is present, the supported is as well
         assert!(model.state.implies(s.presence, t.presence));
 
-        model.enforce(eq(s.start, t.start), [s.presence]);
-        model.enforce(eq(s.end, t.end), [s.presence]);
+        //model.enforce(eq(s.start, t.start), [s.presence]);
+        //model.enforce(eq(s.end, t.end), [s.presence]);
+        // Relaxed constraints in the encoding for chronicles coming from an acting system,
```

Comment:
This changes the model that is used for all planners, including e.g. the HDDL planner. Such a relaxation should be opt-in.

Reply:
I understand, this should be made optional with e.g. an env variable.

Reply:
This should resolve the problem:

https://github.com/plaans/aries/blob/a166397a6dbf6929c2af9134be87a021f14b38a6/planning/planners/src/encode.rs#LL24C1-L31C81

and

https://github.com/plaans/aries/blob/a166397a6dbf6929c2af9134be87a021f14b38a6/planning/planners/src/encode.rs#LL280C9-L289C1